message-hub/README.md

```diff
@@ -0,0 +1,174 @@
+and follow the link to the connection information.
+![JDBC connect dropdown](db2_conn.jpg)
+1. Follow the instructions to install the Db2 JDBC driver for your platform if needed.
+![Install JDBC driver](db2_jdbc_driver.jpg)
+1. Copy ther JDBC connection string as you will need it to configure the
+kafka-connect-jdbc sink connector.
+![JDBC connection string](db2_jdbc.jpg)  
+
+# Setting up Apache Kafka Connect
+
+1. Download [Apache Kafka](https://kafka.apache.org/downloads) - this tutorial used Apache Kafka version 1.0.0 for Scala 2.11.0 (`kafka_2.11-1.0.0.tgz`).
+1. Extract the Apacke Kafka installation.
+1. Create a `connect-standalone.properties` file.
+1. Add these properties, substituting values from your IBM Message Hub service credentials where appropriate.
+
+```properties
+# Substitute your server endpoints as shown in your service credentials
+bootstrap.servers=kafka04-prod02.messagehub.services.eu-gb.bluemix.net:9093,\
+kafka03-prod02.messagehub.services.eu-gb.bluemix.net:9093,\
+kafka05-prod02.messagehub.services.eu-gb.bluemix.net:9093,\
+kafka02-prod02.messagehub.services.eu-gb.bluemix.net:9093,\
+kafka01-prod02.messagehub.services.eu-gb.bluemix.net:9093
+
+# Message producer connection configuration, substitute your username and password
+producer.sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username="username" password="password";
+producer.security.protocol=SASL_SSL
+producer.sasl.mechanism=PLAIN
+producer.ssl.protocol=TLSv1.2
+producer.ssl.enabled.protocols=TLSv1.2
+producer.ssl.endpoint.identification.algorithm=HTTPS
+
+# Message consumer connection configuration, substitute your username and password
+consumer.sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username="username" password="password";
+consumer.security.protocol=SASL_SSL
+consumer.sasl.mechanism=PLAIN
+consumer.ssl.protocol=TLSv1.2
+consumer.ssl.enabled.protocols=TLSv1.2
+consumer.ssl.endpoint.identification.algorithm=HTTPS
+
+# Substitue with the path to the file you want to use to store offset values
+offset.storage.file.filename=offsetFile
+
+# Kafka connect message configuration
+key.converter=org.apache.kafka.connect.json.JsonConverter
+value.converter=org.apache.kafka.connect.json.JsonConverter
+key.converter.schemas.enable=true
+value.converter.schemas.enable=true
+internal.key.converter=org.apache.kafka.connect.json.JsonConverter
+internal.value.converter=org.apache.kafka.connect.json.JsonConverter
+```
+
+# Configuration for publishing messages from Cloudant to Message Hub using Apache Kafka and kafka-connect-cloudant
+
+1. Download the [`kafka-connect-cloudant` jar from Maven central](TODO insert link when available) - this tutorial used `kafka-connect-cloudant-0.100.0-kafka-1.0.0.jar`.
+1. Create a `connect-cloudant-source.properties` file.
+1. Add these properties, substituting your details where appropriate.
+
+```properties
```

Comment:
You can indent lines 98 to 121 to match up with the indent of step 3.

Reply:
Done in 60970d1ba8e31da144cc94261b4b4c7867092d90